app/queries/claims/claims_query.rb

```diff
@@ -48,4 +49,10 @@ def status_condition(scope)
 
     scope.where(status: params[:statuses])
   end
+
+  def academic_year_condition(scope)
+    return scope if params[:academic_year_ids].blank?
+
+    scope.where(claim_window_id: Claims::ClaimWindow.select(:id).where(academic_year_id: params[:academic_year_ids]))
+  end
```

Comment:
I believe this works too. I'm not sure which is objectively better though.

A join is great with lesser records, but a subquery also works well.

```suggestion
  def academic_year_condition(scope)
    return scope if params[:academic_year_ids].blank?

    scope.joins(:academic_year).where(academic_years: { id: params[:academic_year_ids] })
  end
```